README.md

```diff
@@ -1,30 +1,44 @@
-# Hydrogen template: Skeleton
+# Hydrogen with Sentry
 
-Hydrogen is Shopify’s stack for headless commerce. Hydrogen is designed to dovetail with [Remix](https://remix.run/), Shopify’s full stack web framework. This template contains a **minimal setup** of components, queries and tooling to get started with Hydrogen.
+This is a demo for getting started with Hydrogen and Sentry. It's based on the Hydrogen's [default template](https://github.com/Shopify/hydrogen/tree/main/templates/skeleton) with TypeScript.
 
-[Check out Hydrogen docs](https://shopify.dev/custom-storefronts/hydrogen)
-[Get familiar with Remix](https://remix.run/docs/en/v1)
 
-## What's included
+[Sentry Remix SDK Docs](https://docs.sentry.io/platforms/javascript/guides/remix/)
 
-- Remix
-- Hydrogen
-- Oxygen
-- Shopify CLI
-- ESLint
-- Prettier
-- GraphQL generator
-- TypeScript and JavaScript flavors
-- Minimal setup of components and routes
+[Hydrogen Docs](https://shopify.dev/custom-storefronts/hydrogen)
+
+[Remix Docs](https://remix.run/docs/en/main)
 
 ## Getting started
 
 **Requirements:**
 
 - Node.js version 16.14.0 or higher
 
+**Setup:**
+
+1. `npm install`
+2. Place your DSN in `entry.client.tsx` and `entry.server.tsx`
+
+```ts
+Sentry.init({
+  // TODO: replace with your Sentry DSN
+  dsn: 'SENTRY_DSN',
+  // ...
+});
+```
+3. Place your Sentry Organization Slug, Project Slug, and Auth Token in `.env`
+This is required to create Sentry releases and upload source maps.
```

Comment:
Maybe we can link to docs for this? https://docs.sentry.io/platforms/javascript/guides/remix/sourcemaps/#2-create-a-release-and-upload-source-maps